lib/audited/version.rb

```diff
@@ -1,3 +1,3 @@
 module Audited
-  VERSION = "4.9.4"
+  VERSION = "4.9.5"
```

Comment:
we should set a version number to be major, like 5.0.0-ft or something.

Reply:
If we decide to fix tests then yeah, I agree